eslint.js

```diff
@@ -3,18 +3,42 @@
 var config = {
   "root": true,
   "extends": ["eslint:recommended", "prettier"],
+  "plugins": [
+      "node"
+  ],
+  "parserOptions": {
+    "ecmaVersion": 2018
+  },
   "env": {
-    "node": true
+    "node": true,
+    "es2017": true
   },
   "rules": {
+    "node/no-unsupported-features/es-syntax": [2, {"version": ">=0.10"}],
```

Comment:
think maybe you slipped in a `0.` too much here if you are now targeting v10

```diff
-    "node/no-unsupported-features/es-syntax": [2, {"version": ">=0.10"}],
+    "node/no-unsupported-features/es-syntax": [2, {"version": ">=10"}],
```

Reply:
@jimmywarting no this is by design.